Cargo.lock

```diff
@@ -151,13 +151,68 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "constant_time_eq"
-version = "0.1.3"
+version = "0.1.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "crossbeam"
-version = "0.3.2"
+version = "0.7.2"
```

Comment:
It'd be nice to see if we can rely on just one of these subcrates of crossbeam rather than `crossbeam` and all of these other crates. (Not blocking.)